src/vscode-bicep/src/language/client.ts

```diff
@@ -6,28 +6,30 @@
~import { getLogger } from "../utils/logger";
~import {
~  callWithTelemetryAndErrorHandlingSync,
~  IActionContext,
~  parseError,
~} from "vscode-azureextensionui";
~import { ErrorAction, Message, CloseAction } from "vscode-languageclient/node";
~
~const dotnetRuntimeVersion = "3.1";
 const packagedServerPath = "bicepLanguageServer/Bicep.LangServer.dll";
 
 export async function launchLanugageServiceWithProgressReport(
-  context: vscode.ExtensionContext
+  context: vscode.ExtensionContext,
+  outputChannel: vscode.OutputChannel
 ): Promise<void> {
   await vscode.window.withProgress(
     {
       title: "Launching Bicep language service...",
       location: vscode.ProgressLocation.Notification,
     },
-    async () => await launchLanguageService(context)
+    async () => await launchLanguageService(context, outputChannel)
   );
 }
 
 async function launchLanguageService(
-  context: vscode.ExtensionContext
+  context: vscode.ExtensionContext,
```

Comment:
>shell [](start = 15, length = 5)

Just curious - why was it needed?

Reply:
Oops, this was me trying to copy the ARM extension to figure out why my changes weren't working. I'll revert it.